content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/fefo.rst

```diff
@@ -0,0 +1,126 @@
+============
+FEFO removal
+============
+
+The *First Expired, First Out* (FEFO) removal strategy targets products for removal based on their
+assigned removal dates.
+
+.. seealso::
+   :doc:`Other removal strategies <removal>`
+
+.. _inventory/warehouses_storage/removal-date:
+
+Removal date
+============
+
+Products **must** be removed from inventory before their *removal date*, which is set as a certain
+number of days before the product's *expiration date*.
+
+The user sets this number of days by navigating to the product form's :guilabel:`Inventory` tab.
+Under the :guilabel:`Traceability` section, ensure the :guilabel:`Tracking` field is set to either
+:guilabel:`By Lots` or :guilabel:`By Unique Serial Number`.
+
+Next, select the :guilabel:`Expiration Date` option, which makes the :guilabel:`Removal Date` field
+(and other date fields) appear.
+
+.. important::
+   The :guilabel:`Lots and Serial Numbers` and :guilabel:`Expiration Dates` features **must** be
+   enabled in :menuselection:`Inventory app --> Configuration --> Settings` to track expiration
+   dates.
+
+The expiration date of a product is determined by adding the date the product was received to the
+number of days specified in the :guilabel:`Expiration Date` field of the product form.
+
+The removal date takes this expiration date, and subtracts the number of days specified in the
+:guilabel:`Removal Date` field of the product form.
+
+.. seealso::
+   :doc:`Expiration dates <../../product_management/product_tracking/expiration_dates>`
+
+.. example::
+   In the :guilabel:`Inventory` tab of the product, `Egg`, the following :guilabel:`Dates` are set
+   by the user:
+
+   - :guilabel:`Expiration Date`: `30` days after receipt
+   - :guilabel:`Removal Date`: `15` days after expiration date
```

Comment:
think it's before instead of after here, according to the screenshot?
```suggestion
   - :guilabel:`Removal Date`: `15` days before expiration date
```